### DISCUSSION TITLE
How can i get Spotify Client ID

### DISCUSSION THREAD

**Discussion by mustcodes** [community/community#139492] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hello guys. How can i get Spotify Client ID?

**Comment by dusova** [comment, marked as answer]
✔️ Spotify app ClientID is the ID of the app you created on Spotify. This ID is used to identify your app on Spotify. You can create your app and get the ClientID from the Spotify Developers page. 

https://developer.spotify.com/

**Comment by Henryalpha1** [comment]
I had the same question. You just log into the Spotify Developer Dashboard, create an app, and your Client ID shows up right away. I usually use [Spotify Craccato](https://spotifycraccato.com/), and it’s great how smooth and feature-packed the experience feels compared to other apps.
